### ISSUE TITLE
Pelican can't import/register encrypt_content

### ISSUE THREAD

**Issue by AlFontal** [mindcruzer/pelican-encrypt-content#16]
Hi! Thanks for a very useful feature which I really want to get working!

I am quite sure I am doing this wrong, but how am I supposed to organize the plugin folders so that (1) I don't get the `    AttributeError: module 'pelican-encrypt-content' has no attribute 'register'` error or (2) a `ModuleNotFoundError` when generating the pelican content?

**Comment by mindcruzer** [comment]
@AlFontal based on the error message it looks like you've copied the entire `pelican-encrypt-content` folder instead of just the `encrypt_content` folder (ie. /pelican-encrypt-content/**encrypt_content**). Try doing that instead and make sure you set `PLUGINS = ['encrypt_content']`.

**Comment by banality13** [comment]
Hi, thanks for the plugin as well ^^ 
Having the same issue here, but @mindcruzer 's answer doesn't seem to solve the problem. I tried to install another plugin on the side to make sure. 

Here's my folder structure : 
```
├── content
├── output
├── plugins
│   ├── encrypt_content
│   │   └──encrypt_content.py and the other files
│   └── i18n_subsites
```

And my `pelicanconf.py` : 
```python
# [...]
PLUGIN_PATHS = ['./plugins']
PLUGINS = ['encrypt_content', 'i18n_subsites']
# [...]
```

A quick `make` shows me that the `i18n_subsites` module is working, but I am getting the same error as @AlFontal for `encrypt_content` : 
```
ERROR: Cannot register plugin `encrypt_content`
  | module 'encrypt_content' has no attribute 'register'
```

**Comment by mindcruzer** [comment]
I just found out that in Pelican 4.5 the plugin discovery process changed. I will push an update shortly.

**Comment by mindcruzer** [comment]
I've pushed a `pelican-4.5` branch with changes that will support Pelican namespace packages for plugins. If you're using Pelican 4.5 I'd appreciate it if you give it a try and let me know if it works for you.

**Comment by mindcruzer** [comment]
Guys I just realized that the instructions were misleading, which would have lead to the error you were getting. I've merged the `pelican-4.5` branch, which has clearer instructions.

**Comment by banality13** [comment]
Yep, working with namespace install ! Thanks a lot ! :tada:
